asset.go

```diff
@@ -0,0 +1,16 @@
+package open_asset_model
+
+type Asset interface {
+	GetAssetType() string
```

Comment:
agree w/ @fruizrob. AssetType is an alias for string

Also, rename to AssetType(). 

Reference: https://go.dev/doc/effective_go#Getters

Reply:
Renaming the interface to AssetType results in a compile error, because of `type AssetType string`. How do you want me to resolve this?